### DISCUSSION TITLE
Thick and thin plugins

### DISCUSSION THREAD

**Discussion by maxpain** [k8snetworkplumbingwg/multus-cni#885] [Q&A] [answered]
Hi. What is the difference between thick and thin plugins? I didn't find anything in the documentation about this.

**Comment by maiqueb** [comment, marked as answer]
[This section](https://github.com/k8snetworkplumbingwg/multus-cni#thin-plugin-vs-thick-plugin) was added to the README, attempting to answer your question @maxpain . 

Does it help ?
